### PR TITLE
Chunks predictions

### PR DESCRIPTION
**Pull request by gavrin-s**

### CODE REVIEW

File: rest_api.py [hunk 1]
```diff
@@ -53,11 +52,11 @@ def predict():
...
     df[logs] = df[logs].applymap(np.log)
     df[sqrts] = df[sqrts].applymap(np.sqrt)
-    X = df.values.reshape(1, -1)
+    X = df.values.reshape(-1, len(columns))
     X_scaled = scaler.transform(X)
 
-    prediction = np.round(model.predict(X_scaled)).astype(np.int8)[0]
-    return jsonify({"prediction": '{}'.format(prediction)})
+    prediction = np.round(model.predict(X_scaled)).astype(np.int8)
+    return jsonify({"predictions": '{}'.format(prediction)})
```
Comment (denisvmedyantsev) [inline rest_api.py]: It's redundant to use object with one field "predictions" just to wrap an array. As well as use construction like "'{}'.format()".

File: specification.yaml [hunk 2]
```diff
@@ -7,13 +7,41 @@ servers:
...
-      description: File for rediction wine quality
+      description: Prediction wine quality
       requestBody:
-        description: Input params in json
+        description: List of jsons
         content:
           application/json:
             schema:
-              type: object
+              type: array
```
Comment (denisvmedyantsev) [inline specification.yaml:16]: What fields are required?

File: Untitled.ipynb [hunk 3]
```diff
@@ -0,0 +1,197 @@
+{
```
Comment (denisvmedyantsev) [inline Untitled.ipynb]: Use meaningful file names.

**Review by denisvmedyantsev — CHANGES_REQUESTED** [review]

File: specification.yaml (hunk 2 above)
Comment (gavrin-s) [reply specification.yaml:16]: This field is array of integers.

File: Untitled.ipynb (hunk 3 above)
Comment (gavrin-s) [reply Untitled.ipynb]: Forgot to delete

File: specification.yaml (hunk 2 above)
Comment (gavrin-s) [reply specification.yaml:16]: In json.

File: rest_api.py [hunk 4]
```diff
@@ -53,11 +52,11 @@ def predict():
...
     df[logs] = df[logs].applymap(np.log)
     df[sqrts] = df[sqrts].applymap(np.sqrt)
-    X = df.values.reshape(1, -1)
+    X = df.values.reshape(-1, len(columns))
     X_scaled = scaler.transform(X)
 
-    prediction = np.round(model.predict(X_scaled)).astype(np.int8)[0]
-    return jsonify({"prediction": '{}'.format(prediction)})
+    prediction = np.round(model.predict(X_scaled)).astype(np.int8)
+    return jsonify('{}'.format(prediction))
```
Comment (denisvmedyantsev) [inline rest_api.py]: Try to explain the necessity of the '{}'.format(prediction)

File: specification.yaml (hunk 2 above)
Comment (denisvmedyantsev) [reply specification.yaml:16]: Which of item properties are required?

**Review by denisvmedyantsev — CHANGES_REQUESTED** [review]

File: specification.yaml (hunk 2 above)
Comment (gavrin-s) [reply specification.yaml]: Here are not properties.

File: rest_api.py (hunk 4 above)
Comment (gavrin-s) [reply rest_api.py]: The string is converted to json , the array is not.

File: specification.yaml (hunk 2 above)
Comment (gavrin-s) [reply specification.yaml:16]: This comment is not on the last commit. This has already been changed.

**Review by olelukoje — APPROVED** [review]

**Review by denisvmedyantsev — APPROVED** [review]

**Pull request merged**